po/es.po

```diff
@@ -872,11 +872,11 @@ msgstr "Send y Sync"
 
 #: src/SUMMARY.md:274
 msgid "Send"
-msgstr "Enviar"
+msgstr "Send"
```

Comment:
I restarted a conversation in https://github.com/google/comprehensive-rust/discussions/1184#discussioncomment-7243894 , because I'm not sure what the guidance is, whether to use English or Spanish for these.

Reply:
In this particular case, "Send" (or rather `Send`) is correct since it's the name of a trait in the standard library. So it's like `String` or `HashMap`.

I think that's also what you concluded in #1184.

Reply:
Let me resolve this conversation since using `Send` is correct here.